### ISSUE TITLE
Wrong Version of mdEditor Pushed into Production

### ISSUE THREAD

**Issue by dwalt** [adiwg/mdEditor#565]
It appears a development version of mdEditor has been pushed to production prematurely. Contact External Identifier appears in mdEditor, but does not work properly or write to mdJSON. There is also an error in the interface for display of number of records. I believe this was an earlier development version that Tim had been working on, which has since advanced with a fix of the interface and a write to mdJSON. There is a possibility that other changes may have been stepped on by this version. Noticed 7/13/23.

**Comment by hmaier-fws** [comment]
The "External Identifiers" card was accidentally pushed out to the production version before the feature was fully implemented. It does seem to work properly in the develop version and the issue should be resolved once develop is merged into the master branch.
